lib/data/benefit_cap_data.yml

```diff
@@ -1,4 +1,21 @@
 ---
-single: 350
-couple: 500
-parent: 500
+rates:
+  single: 350
+  couple: 500
+  parent: 500
+benefits:
```

Comment:
@chrisroos These were added as part of the modeling phase and later used.. [here](https://github.com/alphagov/smart-answers/blob/benefit-cap-calculator-refactoring/test/unit/calculators/benefit_cap_calculator_configuration_test.rb#L11)